rebel-extension/src/components/CanvasTokenManager.jsx

```diff
@@ -56,30 +56,30 @@ const CanvasTokenManager = () => {
     };
 
     return (
-        <div className="p-6 max-w-md rounded-lg shadow-md text-center"> {/* Removed mx-auto so that the background color wouldn't show through */}
+        <div className="p-6 max-w-md rounded-lg shadow-md text-center CVTDiv"> {/* Removed mx-auto so that the background color wouldn't show through */}
             {/* Input Field for Canvas Token */}
-            <div className="relative">
+            <div className="relative CVTTextDiv">
                 <input
                     type={showToken ? "text" : "password"} // Toggles between text and password visibility
                     placeholder="Enter Canvas Token"
                     value={canvasToken}
                     onChange={(e) => setCanvasToken(e.target.value)}
-                    className="border p-2 rounded w-full text-gray-700"
+                    className="border p-2 rounded canvasInputText "
                 />
 
                 {/* Toggle Visibility Button */}
                 <button
                     onClick={() => setShowToken(!showToken)}
-                    className="absolute right-2 top-2"
+                    className="canvasShowBT"
                 >
                     {showToken ? "🙈 Hide" : "👁 Show"}
-                </button>
+                </button>r
```

Comment:
```suggestion
                </button>
```
Please remove this r. It is adding an extra r in both places the CanvasTokenManager is visible. 
![image](https://github.com/user-attachments/assets/49ca7598-22b1-4aa4-9144-942f7037f74a)

![image](https://github.com/user-attachments/assets/50d94c64-24df-4551-950c-eaed29b0f685)

Reply:
Thank you for pointing this out! Fixed in latest commit.